### PR TITLE
[PB-441]: fix/usage flashes when uploading

### PR DESCRIPTION
**Pull request by rafijv**
- Only reload the used space once when multiple files are uploaded
- Show progress when uploading any kind of item

### CODE REVIEW

File: src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts [hunk 1]
```diff
@@ -108,6 +109,7 @@ export const uploadFolderThunk = createAsyncThunk<void, UploadFolderThunkPayload
             .unwrap()
             .then(() => {
               alreadyUploaded += level.childrenFiles.length;
+              alreadyUploaded += 1;
```
Comment (sg-gs) [inline src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts:112]: why?

File: src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts [hunk 2]
```diff
@@ -313,9 +323,10 @@ function countItemsUnderRoot(root: IRoot): number {
   while (queueOfFolders.length > 0) {
     const folder = queueOfFolders.shift() as IRoot;
 
-    count += folder.childrenFiles?.length ?? 0;
+    count += folder.childrenFiles.length;
 
     if (folder.childrenFolders) {
+      count += folder.childrenFolders.length;
```
Comment (CandelR) [inline src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts:329]: Folders are not uploaded to the network, won't this change the upload % when there are many subfolders to upload?

Comment (rafijv) [reply src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts:329]: This is what is required in the second part of the issue.

File: src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts (hunk 1 above)
Comment (rafijv) [reply src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts:112]: When uploading empty folders are counted in the upload progression

**Review by CandelR — APPROVED** [review]

**Pull request merged**